ansible/roles/mongodb-backup/meta/main.yml

```diff
@@ -0,0 +1,2 @@
+dependencies:
```

Comment:
dependency will be handled by the include_role right?

Reply:
the include role does not have the meta. so this is ok to define here.